docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -266,7 +266,7 @@ First install Grafana, and then add Mimir as a Prometheus data source.
 
    You should be able to query metrics in [Grafana Explore](http://localhost:3000/explore),
    as well as create dashboard panels by using your newly configured `Mimir` data source.
-   For more information, see [Monitor Grafana Mimir]({{< relref "/docs/mimir/v2.5.x/operators-guide/monitor-grafana-mimir/" >}}).
+   For more information, see [Monitor Grafana Mimir]({{< relref "/docs/mimir/latest/operators-guide/monitor-grafana-mimir/" >}}).
```

Comment:
@krajorama, this relates to #3736, correct?

Reply:
yes, correct

Reply:
@jdbaldry, what do you recommend wrt keeping this value verbatim in special cases such as this one?